apps/pie-microsite/src/assets/styles/components/_tabs.scss

```diff
@@ -1,16 +1,16 @@
 @use '../fozzie-config' as f;
+@use '../variables' as v;
 
 $tabs-link-padding:                f.spacing(d);
 
 .c-tabs {
   z-index: f.zIndex(mid);
   font-weight: f.$font-weight-bold;
-  padding: 0;
 
   ul {
     $tabs-padding: 64px - 16px; // To match grid-template-rows from layout file.
```

Comment:
just noticed - if there is a value here defined elsewhere then can we make it a shared variable please?

Reply:
This isn't actually needed at all in this file anymore